drivers/wifi/infineon/Kconfig.airoc

```diff
@@ -37,6 +54,17 @@ config AIROC_WIFI_CUSTOM
 	  user must to provide path to FW, CLM and NVRAM for
 	  custom or vendor CYW43xx modules.
 
+if AIROC_WIFI_BUS_SPI
+
+config AIROC_WIFI_BUS_SPI_DATA_IRQ_MUX
+	bool "Data and IRQ GPIO is shared"
+	help
+		Enable logic when the SPI data GPIO is shared
+		with the host wake interrupt, which requires
+		switching GPIO modes for that pin.
```

Comment:
Typically the help text is aligned with two spaces, please change this.

Reply:
Fixed.